ecs/variables.tf

```diff
@@ -36,6 +36,10 @@ variable "consignment_export" {
   default = false
 }
 
+variable "jenkins" {
+  default = false
```

Comment:
Maybe a description here just to be clear what this variable means.

Reply:
I've added one in